mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job.go.erb

```diff
@@ -789,7 +789,7 @@ func resourceDataflowFlexJobTypeCustomizeDiff(_ context.Context, d *schema.Resou
 				continue
 			}
 
-			if field != "labels" && field != "terraform_labels" && d.HasChange(field) {
```

Comment:
Shouldn't this be an OR rather than an AND? Either `terraform_labels` or `labels` could safely change without recreating the resource, as long as `effective_labels` doesn't. (same below)

Reply:
(I might be missing something though)

Reply:
Here `!=` is used, so should use `AND`?

Reply:
For other resources, both `labels` and `terraform_labels` have `ForceNew: true`. Should we match with other resources?

Reply:
Oh, you're right! I misread the conditional. Matching other resources seems reasonable, good point. We'd remove `field != "terraform_labels"` for that?

Reply:
Sorry, for other resources, both `labels` and `effective_labels` have `ForceNew: true`. `terraform_labels` doesn't have  `ForceNew: true`

Reply:
I realize that right now when `terraform_labels` changes, it doesn't recreate the resource, but it sends the update request.

I am going to make change to not send the update request when `terraform_labels` changes.